apps/backoffice-v2/src/common/components/organisms/Header/hooks/useNavbarLogic/useNavbarLogic.tsx

```diff
@@ -0,0 +1,68 @@
+import { useFiltersQuery } from '@/domains/filters/hooks/queries/useFiltersQuery/useFiltersQuery';
+import { useFilterId } from '@/common/hooks/useFilterId/useFilterId';
+import { useCallback, useMemo } from 'react';
+import { Building, Goal, Users } from 'lucide-react';
+import { TRoutes, TRouteWithChildren } from '@/Router/types';
+
+export const useNavbarLogic = () => {
+  const { data: filters } = useFiltersQuery();
+  const filterId = useFilterId();
+  const individualsFilters = useMemo(
+    () => filters?.filter(({ entity }) => entity === 'individuals'),
+    [filters],
+  );
+  const businessesFilters = useMemo(
+    () => filters?.filter(({ entity }) => entity === 'businesses'),
+    [filters],
+  );
+  const navItems = [
+    {
+      text: 'Businesses',
+      icon: <Building size={20} />,
+      children:
+        businessesFilters?.map(({ id, name }) => ({
+          filterId: id,
+          text: name,
+          href: `/en/case-management/entities?filterId=${id}`,
+          key: `nav-item-${id}`,
+        })) ?? [],
+      key: 'nav-item-businesses',
+    },
+    {
+      text: 'Individuals',
+      icon: <Users size={20} />,
+      children:
+        individualsFilters?.map(({ id, name }) => ({
+          filterId: id,
+          text: name,
+          href: `/en/case-management/entities?filterId=${id}`,
+          key: `nav-item-${id}`,
+        })) ?? [],
+      key: 'nav-item-individuals',
+    },
+    {
+      text: 'Transaction Monitoring',
```

Comment:
Same point for Transaction Monitoring - I'm guessing not all our customers will have it

Reply:
We want to show this to all of our customers.